src/browser/modules/Stream/CypherFrame/VisualizationView.tsx

```diff
@@ -154,7 +157,7 @@ export class Visualization extends Component<any, VisualizationState> {
     return new Promise<any>((resolve, reject) => {
       this.props.bus &&
         this.props.bus.self(
-          CYPHER_REQUEST,
+          ROUTED_CYPHER_READ_REQUEST,
```

Comment:
Risk for consistency issues, as the write query to would be on a different core from the read query and it might not have gotten the update yet. Should be handled by using the same session through something called "bookmarks"

Reply:
Leaving this as is for that reason